### PR TITLE
Starting to add edits to address changes in JSON-LD 1.1.

### PR DESCRIPTION
**Pull request by aphillips**
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/string-meta/pull/48.html" title="Last updated on Jul 16, 2020, 6:10 PM UTC (063ee2f)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/string-meta/48/65bccf4...063ee2f.html" title="Last updated on Jul 16, 2020, 6:10 PM UTC (063ee2f)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -209,7 +209,10 @@ <h3 id="string_specific_language">String-specific language information</h3>
...
 
-<p> Low-level support for natural language string metadata is widespread because the use of metadata for storage and interchange of the language of data values is long-established and widely supported in the basic infrastructure of the Web. This includes language attributes in [[XML]] and [[HTML]]; string types in schema languages (e.g. [[xmlschema11-2]]) or the various RDF specifications including [[JSON-LD]]; or protocol- or document format-specific provisions for language.</p>
+<p>Low-level support for natural language string metadata is widespread because the use of metadata for storage and interchange of the language of data values is long-established and widely supported in the basic infrastructure of the Web. This includes language attributes in [[XML]] and [[HTML]]; string types in schema languages (e.g. [[xmlschema11-2]]) or the various RDF specifications including [[JSON-LD]]; or protocol- or document format-specific provisions for language.</p>
+
+<p class="advisement" id="bp-use_jsonld_i18n_namespace"><a class="self" href="#bp-use_jsonld_i18n_namespace">&#x200B;</a>Specify the use of [[JSON-LD]] 1.1 <code>i18n namespace</code> feature for RDF literals.</p>
+
```
Comment (iherman) [inline index.html:215]: This is not a "JSON-LD 1.1 i18n namespace". It is a generic namespace defined for RDF which has been defined in the JSON-LD document (for all the known reasons) but can be used in any RDF serialization _including_ JSON-LD. Although I know this is not the subject of this document, but the same properties could be used in Turtle, for example.

Maybe:

> Specify the use of the <code>i18n namespace</code> feature for RDF literals, as defined in the [[JSON-LD]] 1.1 document

File: index.html [hunk 2]
```diff
@@ -228,8 +231,10 @@ <h3 id="string_specific_direction">String-specific directional information</h3>
...
-<p class="issue">Schema languages, such as the RDF suite of specifications, have no in-built mechanism for associating base direction metadata with natural language string values.</p>
-<p class="issue">There is no built-in attribute for base direction in [[JSON-LD]]. There needs to be a corresponding built-in attribute (e.g. a <code class="kw" translate="no">dir</code>) or de facto convention for indicating document-level base direction.</p>
+<div class="note">
+	<p>Some schema languages, such as the RDF suite of specifications, have no built-in mechanism for associating base direction metadata with natural language string values. It is up to specifications that use these specifications to define structures and adopt best practices that result in clean interchange of language and direction metadata.</p>
+    <p>For example, [[JSON-LD]] provides a document-level base direction using the <code>@Context</code> mechanism and defines the <code>i18n namespace</code> as a non-standard RDF datatype for setting either/both language and base direction of a string.</p>
```
Comment (iherman) [inline index.html]: See my comment above. RDF does not have a built-in mechanism, but now has an 'extension', as proposed in the JSON-LD specification, to use i18n specific terms. That can be used by any user of RDF, regardless of serialization...

File: index.html (hunk 1 above)
Comment (aphillips) [reply index.html:215]: Thanks. I adopted this text.

File: index.html (hunk 2 above)
Comment (aphillips) [reply index.html]: See if the new revision captures it. 

I know I need to do more work here--adding documentation and examples later in the doc, etc.

File: index.html [hunk 3]
```diff
@@ -223,17 +223,14 @@ <h3 id="string_specific_language">String-specific language information</h3>
...
-    <p>For example, [[JSON-LD]] provides a document-level base direction using the <code>@Context</code> mechanism and defines the <code>i18n namespace</code> as a non-standard RDF datatype for setting either/both language and base direction of a string.</p>
+    <p>Some schema languages, such as the RDF suite of specifications, have no built-in mechanism for associating base direction metadata with natural language string values. It is up to specifications that use these specifications to define structures and adopt best practices that result in clean interchange of language and direction metadata.</p>
+    <p>For example, [[JSON-LD]] provides a document-level base direction using the <code>@Context</code> mechanism and defines the <code>i18n namespace</code> as an extension of existing RDF datatypes which can be used to set the language and/or base direction of string values.</p>
```
Comment (iherman) [inline index.html]: Tiny bit: it should be <code>@context</code> not <code>@Context</code>

**Review by iherman — APPROVED** [review]

**Pull request closed without merging**